Document/0x05g-Testing-Network-Communication.md

```diff
@@ -169,7 +169,7 @@ If a certificate pinning validation check has failed, the following event will b
 I/X509Util: Failed to validate the certificate chain, error: Pin verification failed
 ```
 
-#### Static Analysis
+##### Static Analysis
```

Comment:
There should be one static analysis section with #### Static Analysis and then,
##### Trust Manager
##### Cordova Applications
etc.